packages/SwingSet/test/virtualObjects/test-representatives.js

```diff
@@ -24,132 +24,6 @@ function slot0(iface, kid) {
   };
 }
 
-test('virtual object representatives', async t => {
```

Comment:
I see a whole big test was deleted.. is this functionality now covered by something else?

Reply:
No, this test went away because the set of weird edge cases whose proper handling it tested are no longer expressible in the API.  A corollary to that is that the test itself is no longer expressible.